packages/client/hmi-client/src/assets/css/theme/_asset.scss

```diff
@@ -0,0 +1,95 @@
+.asset {
```

Comment:
Could you add a header comment explaining the purpose of this stylesheet.

Also, it shouldn't be in the `/theme` folder, that's for _PrimeVue_, you can place this file  next `css/styles.scss` and import it (like `reset.css`).